core/frontend/src/ModelState.ts

```diff
@@ -178,6 +178,7 @@ export abstract class GeometricModelState extends ModelState implements Geometri
         modelId: this.id,
         tilesetToDbTransform: this.jsonProperties.tilesetToDbTransform,
         classifiers: undefined !== spatialModel ? spatialModel.classifiers : undefined,
+        scalablemeshProps: undefined !== spatialModel ? spatialModel.scalablemeshProps : undefined,

```

Comment:
You should add this scalableMeshProps to the "if (rdSourceKey)" section above. The rdSourceKey will have priority when present and scalableMeshProps should also be supported in this workflow.

Reply:
Done! See 3f8952f